scripts/sct_propseg.py

```diff
@@ -416,6 +415,9 @@ def generate_qc(fn_in, fn_seg, args, path_qc):
             use_optic = False
     if "-init" in arguments:
         init_option = float(arguments["-init"])
+        if (init_option < 0):
+            sct.log.error('Error! Negative number asigned! Please provide a valid value for "-init"')
```

Comment:
There's a certain command-line error pattern in msct_parter which makes such errors not caught by Sentry. You should use that format (sorry, it's not documented).

Reply:
"msct_parser" you mean? can you document the format please?

Reply:
Look at `server_log_handler()` in sct_utils. The Sentry logging handler is monkey-patched to discard certain (user PEBKAC) error messages.